train_network.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+import numpy as np
+from accuracy import *
+from DataReader import *
+from DRN import *
+import os
+from dice import *
+
+tf.reset_default_graph()
+directory = os.getcwd()
+
+train_file = os.path.join(directory, 'data', 'train.txt')
+test_file =  os.path.join(directory, 'data', 'val.txt')
+logs = os.path.join(directory, 'logs')
+trainloss = os.path.join(logs, 'train_loss.txt')
+
+
+if os.path.isdir(logs) == False:
```

Comment:
while `== False` technically gives the same result, `not os.path.isdir(logs)` is more pythonic and appropriate.